packages/admin-ui/src/filters/__stories__/filter-advanced.stories.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useEffect } from 'react'
+import type { Meta } from '@storybook/react'
+
+import { FilterDisclosure } from '../filter-disclosure'
+import { FilterPopover } from '../filter-popover'
+
+import { FilterFooter } from '../filter-footer'
+import { FilterListbox } from '../filter-listbox'
+
+import { useFilterMultipleState } from '../filter-multiple/filter-multiple.state'
+import { FilterOptionCheckbox } from '../filter-multiple/filter-option-checkbox'
+
+import { FilterGroup } from '../filter-group'
+import { useFilterGroupState } from '../filter-group.state'
+import { FilterControl } from '../filter-control/filter-control'
+import { useFilterControl } from '../filter-control/filter-control-state'
+import { FilterOptional } from '../filter-control/filter-optional'
+
+export default {
+  title: 'admin-ui/Filters/advanced',
+  component: FilterGroup,
+} as Meta
+
+const list1 = [
+  { label: 'Full', id: '#1' },
+  { label: 'Empty', id: '#2' },
+  { label: 'Half full', id: '#3' },
+  { label: 'Half empty', id: '#4' },
+  { label: 'Unknown', id: '#5' },
+  { label: 'Half empty', id: '#10' },
+]
+
+const list2 = [
+  { label: 'Rio de Janeiro', value: { grade: 10 }, id: '#1' },
+  { label: 'New York', value: { grade: 9 }, id: '#2' },
+  { label: 'Paris', value: { grade: 9 }, id: '#3' },
+  { label: 'Tokyo', value: { grade: 9 }, id: '#4' },
+]
+
+function GenericFilter({ list, label }: { list: any[]; label: string }) {
+  const state = useFilterMultipleState()
+
+  return (
+    <>
+      <FilterDisclosure state={state}>{label}</FilterDisclosure>
+
+      <FilterPopover state={state}>
+        <FilterListbox>
+          {list.map((item) => (
+            <FilterOptionCheckbox {...item} />
+          ))}
+        </FilterListbox>
+        <FilterFooter />
+      </FilterPopover>
+    </>
+  )
+}
+
+export function GroupWithHiddenFilters() {
+  const togState = useFilterControl()
+
+  const filterGroupState = useFilterGroupState({
+    filterStates: [],
+  })
+
+  return (
+    <FilterGroup state={filterGroupState}>
+      <GenericFilter list={list1} label="Important filter" />
+      <GenericFilter list={list2} label="Price" />
+      <GenericFilter list={list1} label="Status" />
+
+      <FilterOptional id="#rare" label="Rare" state={togState}>
```

Comment:
It occurred to me that the label is not being used here, is it still necessary?

Reply:
We are, it's used on the filtercontrol list

Reply:
That's true! Sorry